Tools/scripts/build-jsbsim.sh

```diff
@@ -6,16 +6,20 @@ echo "---------- $0 start ----------"
 
 # don't waste time rebuilding jsbsim if we already have a working copy of it
 # this can save ~10 or minutes on some hardware
-JSBBINARY=~/jsbsim/src/JSBSim
+JSBBINARY=~/jsbsim/build/src/JSBSim
 if [ -f $JSBBINARY ]; then
     echo "$JSBBINARY exists, skipping build. (to force rebuild then remove this file and re-run) "
 else
     echo "$JSBBINARY does not exist, building it in your home folder:"
     cd ~ 
     rm -rf jsbsim
-    git clone https://github.com/tridge/jsbsim.git
+    git clone git://jsbsim.git.sourceforge.net/gitroot/jsbsim/jsbsim
```

Comment:
We should probably clone the JSBSim repository into ArduPilot and clone that.  Using the remote here will lead to random results for people as commits go into that tree.  We could reasonably assume someone updating ArduPilot's clone of JSBSim has made sure we work against it :-)

Reply:
I completely agree with you. I do not have rights for doing that though, so what is the procedure here?

Anyway in the meantime I guess this is good solution since current repo that ArduPilot clones is not working at all and with new repo there is a chance of being broken.